cinch/roles/jenkins_master/tasks/configure.yml

```diff
@@ -23,5 +23,14 @@
     group: jenkins
     state: directory
 
+- name: set jenkins slaveAgentPort
+  template:
+    src: config.xml.j2
```

Comment:
Let's rename this play to indicate that it's doing much more than just setting the slave agent port.